deploy/kicbase/Dockerfile

```diff
@@ -106,6 +106,12 @@ RUN sh -c "echo 'deb https://download.docker.com/linux/ubuntu focal stable' > /e
     apt-key add - < docker.key && \
     clean-install docker-ce docker-ce-cli containerd.io
 
+# install buildkit
```

Comment:
Can you update the comment to mention why we're pulling from this repo as opposed to an official one? It seems suspect.

Reply:
**There is no** official one yet, and I haven't made the request yet...

It's built from https://github.com/afbjorklund/buildkit/tree/debian

Basically I copied the ubuntu packaging for "runc", for "buildkit"

Probably should upload the .dsc and the tarballs to bintray.com ?

Reply:
See also https://github.com/kubernetes/minikube/issues/9552

Reply:
Made a similar hack for podman

Reply:
Does it really need to be dpkg?

Reply:
It doesn't _have_ to be, no. We just preferred using regular system packages, and getting all the software from one place.
But since we moved from ubuntu.com to docker.com for runc/containerd/dockerd - we can get buildkit from github.com

At least until there is an official package available.

Reply:
Adopted the containerd installation script from kind, and adopted to minor quirks in the buildkit release.